src/ovirtapi.js

```diff
@@ -544,9 +544,12 @@ OvirtApi = {
     const url = `${AppConfiguration.applicationContext}/api/`
     return OvirtApi._httpGet({ url })
   },
-  getVm ({ vmId }: VmIdType): Promise<Object> {
+  getVm ({ vmId, additional }: { vmId: string, additional: Array<string> }): Promise<Object> {
```

Comment:
Is the `additional` set somewhere?

Reply:
It's the same `additional` as `getVmsByPage` or `getVmsByCount`.  The saga function `fetchSingleVm()` uses it to do a deep fetch of a single VM, falling back to the 6 REST calls needed in pre-4.2 versions.